models/repo.go

```diff
@@ -254,6 +255,14 @@ func (repo *Repository) ComposeMetas() map[string]string {
 			"user":   repo.MustOwner().Name,
 			"repo":   repo.Name,
 		}
+		switch repo.ExternalTrackerStyle {
+		case "numeric": fallthrough
```

Comment:
I can't remember how exactly `fallthrough` works, but I think it fallthroughs to `case "alphanumeric":` and is not what we intend to?

I suggest just remove this case, let `default` handles it.